examples/binary/artipie.yaml

```diff
@@ -0,0 +1,7 @@
+meta:
+  storage:
+    type: fs
+    path: /var/artipie/configs
+  metrics:
+    type: log
+  layout: flat
```

Comment:
@Sammers21 I'd suggest to remove not required params from example to make it simple. `metrics` are not required (the bug has been fixed), `layout` seems to be `flat` by default

Reply:
@olegmoz, Metrics have been removed. Layout can't be removed, because it would not work without it.

Reply:
@Sammers21 that's sad, hope there is an issue about it